src/combat.ts

```diff
@@ -30,7 +37,83 @@ import {
   Macro as LibramMacro,
   SourceTerminal,
 } from "libram";
-import { maxPassiveDamage, monsterManuelAvailable } from "./lib";
```

Comment:
Brought these methods into this file to remove a circular dependency (since lib uses embezzlers file)